packages/taquito-michelson-encoder/test/tokens/bool.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { BoolToken } from '../../src/tokens/comparable/bool';
+
+describe('Bool token', () => {
+  let token: BoolToken;
+  beforeEach(() => {
+    token = new BoolToken({ prim: 'bool', args: [{ prim: 'int' }], annots: [] }, 0, null as any);
```

Comment:
This doesn't change the behavior of the test, but I think that the args should be empty